ar_module2_8.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+var myDate = (function iffe() {
+  /* eslint-disable no-console */
+  /* eslint-disable no-unused-vars */
+  var privateDate = new Date(); // make this available to both get and set
+  var dd = privateDate.getDate();
+  var mm = privateDate.getMonth() + 1; //January is 0!
+  var yyyy = privateDate.getFullYear();
+
+  if (dd < 10) {
+    dd = '0' + dd;
+  }
+  if (mm < 10) {
+    mm = '0' + mm;
+  }
+
+  var checkDate = mm + dd + yyyy;
+  var formatDate = mm + '/' + dd + '/' + yyyy;
+
+  function setDate(newDate) {  // make a function for milli
+
+    if (newDate == undefined) {
```

Comment:
Always use strict equality `===`. This is something I highly recommend adding to your lint configuration. JavaScript will do type coercion on checks like this, meaning you can get results you don't expect.